task_scheduler/twitch_periodic_tasks.py

```diff
@@ -12,13 +12,14 @@
 from twitchAPI.types import AuthScope
 from .managed_file import ManagedFile
 from .keys import my_app_key, my_app_secret, my_id
+from pys.settings import BASE_DIR
 
 
 def twitch_parse():
     logging.warning("It is time to start the dramatiq task twitch")
     Twitch_model.objects.all().delete()
-    token_tw = "/home/ucsm/Документы/py/pyss/token_tw.pickle"
-    refresh_token_tw = "/home/ucsm/Документы/py/pyss/refresh_token_tw.pickle"
+    token_tw = f"{BASE_DIR}/pyss/token_tw.pickle"
+    refresh_token_tw = f"{BASE_DIR}/pyss/refresh_token_tw.pickle"
```

Comment:
resolved #8 

[?](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)